### PR TITLE
feat: Edit link on operation view mode

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

File: html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig [hunk 1]
```diff
@@ -31,6 +31,13 @@
     </div>
   {% endif %}
 
+  {% if can_edit %}
+    {{ attach_library('common_design_subtheme/node-edit-link') }}
+    <div class="node-edit--link">
+      <a href="{{ path('entity.node.edit_form', {'node': node.id}) }}?destination={{ path('<current>') }}">{{ 'Edit this page'|t }}</a>
```
Comment (rupl) [inline html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig:37]:
```suggestion
      <a href="{{ path('entity.node.edit_form', {'node': node.id}) }}?destination={{ path('<current>') }}"><span class="visually-hidden">{{ 'Edit this page'|t }}</span></a>
```

Marking text as visually-hidden still allows icon to appear on hover without overlap text:

Screen Shot 2022-09-05 at 14 21 56

**Review by rupl — APPROVED** [review]
One idea after looking at it locally. No soldiers ;)

File: html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig (hunk 1 above)
Comment (attiks) [reply html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig:37]: The icon isn't from this patch

Comment (rupl) [reply html/themes/custom/common_design_subtheme/templates/node--operation-tab.html.twig:37]: gotcha, ignore me

**Pull request merged**